masonry/src/text/selection.rs

```diff
@@ -184,8 +184,16 @@ impl<T: Selectable> TextWithSelection<T> {
                         }
                         _ => Handled::No,
                     },
-                    winit::keyboard::Key::Unidentified(_) => todo!(),
-                    winit::keyboard::Key::Dead(_) => todo!(),
+                    winit::keyboard::Key::Unidentified(key) => {
+                        // TODO: figure out correct behavior
+                        tracing::warn!(?key, "got unidentified key, did not handle");
```

Comment:
I suspect that just doing `Handled::No` here is best.

It's bad form to log the keys that a user has input, in case they have e.g. tried to enter a password.